### ISSUE TITLE
Load Previous Scrollbar height

### ISSUE THREAD

**Issue by snewcomer** [ember-infinity/ember-infinity#269]
Setting the proper height for the scrollbar based on how many records could be loaded would greatly improve the scrolling experience and open up scroll top functionality as noted by @StefanFransen.

https://www.vergelijk.nl/notebook/p/8/

**Comment by davidbilling** [comment]
@snewcomer Maybe this is related to my problem? I have a message feed that uses the load previous. I start on the last page I have. But if the last page don't have as many as my "perPage" and does not fill up the page, it won't load the previous page. 

I.E I have 15 records, load 10 per page and start with page 2 that shows the 5 last.

**Comment by snewcomer** [comment]
@davidbilling Yeah this sounds like it is similar to #279, except the opposite way.  Is this your thought as well?

**Comment by davidbilling** [comment]
@snewcomer Yes, it's exactly like #279 in opposite way. But this time I have no idea how to solve it.
